gc/src/gc.rs

```diff
@@ -63,8 +70,15 @@ impl<T: Trace> GcBox<T> {
             let mut st = _st.borrow_mut();
 
             // XXX We should probably be more clever about collecting
```

Comment:
I guess this comment can go now. There's probably still room for improvement, but we're no longer doing the "collect until it stops hurting" strategy; which was mostly a temporary thing.